modules/tts/tts-adapters/tts-adapter-sapinative/src/main/java/org/daisy/pipeline/tts/onecore/impl/OnecoreEngine.java

```diff
@@ -259,8 +263,7 @@ public Collection<Voice> getAvailableVoices() {
 					original.getGender().get()
 				)
 			);
-
-			Logger.info("Voice available : " + original.toString());
+			//Logger.info("Voice available : " + original.toString());
```

Comment:
Either remove, leave as is, or change log level. I don't like if it's commented out.

Reply:
I've removed it.